src/api/libcellml/module/libcellml

```diff
@@ -24,13 +24,15 @@ limitations under the License.
  * \section intro_sec Introduction
  *
  * This is the source code documentation for the libCellML C++ library.
- *
  */
-#include <libcellml/version.h>
-#include <libcellml/model.h>
 #include <libcellml/component.h>
+#include <libcellml/error.h>
 #include <libcellml/import.h>
+#include <libcellml/logger.h>
+#include <libcellml/model.h>
+#include <libcellml/parser.h>
 #include <libcellml/units.h>
 #include <libcellml/variable.h>
+#include <libcellml/version.h>
```

Comment:
Yes, I am definitely happy to consider those includes to be treated as 'external'.